### DISCUSSION TITLE
NAUTOBOT Port

### DISCUSSION THREAD

**Discussion by N0odlez** [mattieserver/netbox-topology-views#337] [General]
Following a discussion from the Nautobot Community. Would you ( @mattieserver ) consider making an additional plugin for Nautobot. It would require a little porting over.

**Comment by mattieserver** [comment]
To be honest, the company I work for has no plans to move to nautobot and i personally also don't use it so the incentive to create and maintain a port is not very high. 

I might take a look at how much is different between the two but should not see this as a promise that I will actually create the port

**Comment by dreng** [comment]
I'm just a co-developer but personally I don't have any relationship with Nautobot and therefore no interest in developing anything for it. My opinion: If there's enough interest then there should be a person in the Nautobot Community who is able to develop a port or join us in order to make this plugin work on both systems. It is already difficult to keep this plugin at an acceptable level for two people in their limited free time. To invest time in developing an open source product that you don't use yourself seems very wrong to me.
